pkg/config/keys.go

```diff
@@ -13,6 +13,7 @@ package config
 import (
 	"github.com/cockroachdb/cockroach/pkg/keys"
 	"github.com/cockroachdb/cockroach/pkg/roachpb"
+	"github.com/cockroachdb/cockroach/pkg/sql/catalog/descpb"
 )
 
 // MakeZoneKeyPrefix returns the key prefix for id's row in the system.zones
```

Comment:
This _function_ (as opposed to a method on the `SystemConfig` type) should thread in a codec.